episodes/introduction.md

```diff
@@ -71,21 +73,19 @@ This lesson will focus primarily on computational reproducibility.
 Like all forms of reproducibility, there are multiple "levels" of reproducibility.
 For this lesson we will focus on "full" reproducibility, meaning that reproducible software environments will:
 
-* Be defined through high level user configuration files.
-* Have machine produced hash level lock files with a full definition of all software in the environment.
-* Specify target computer platforms for all environments solved.
-* Have the resolution and "solving" of a platform's environments be machine agnostic.
-* Have the software packages defined in the environments exist on immutable public package indexes.
+* Be defined through user readable configuration files.
+* Have machine produced files, called lock files, with a full definition of all software in the environment.
+* Have the environment prepared for any computer platform, be machine agnostic.
```

Comment:
These are good, but I think it is relevant to separate out that the platforms are being defined and captured in the information from the fact that different platforms can resolve and manipulate the environments for other platforms. I've added an explicit example

> (e.g. macOS platform machines can solve environments for Linux platforms)

to hopefully clarify this for readers, but this runs the risk of being verbose now. I can further work on revising this a bit.